Automattic/wp-calypso
# WooCommerce Install: Update styles

Brings styles closer to the figma file.

#### Changes proposed in this Pull Request

* Moves styles to a central file
* Establishes a mixing for two-column layout that can be used by onboarding steps.
* Updates form element styles to correspond with the figma design.
* Moves two elements into the woocommerce-install/index file since they're now used by all steps.

#### Testing instructions



- Go to http://calypso.localhost:3000/woocommerce-installation/
- Choose a test site site
- Click "Set up my store"
- Navigate through the onboarding steps and make sure the form elements look like in the figma
- Check that the transfer screen still looks the same.

#### After

<img width="1391" alt="Screen Shot 2022-01-18 at 3 57 28 PM" src="https://user-images.githubusercontent.com/1398304/150037948-383e9f5d-13a8-4f08-8815-30597222bbe9.png">


<img width="1391" alt="Screen Shot 2022-01-18 at 3 57 33 PM" src="https://user-images.githubusercontent.com/1398304/150037956-59bdb0ad-f1fd-4459-9fc1-1763e97c3f50.png">
